lib/foreman_maintain.rb

```diff
@@ -67,9 +79,22 @@ def init_logger
       # argument to Logger.new will not work as expected
       filename = File.expand_path("#{config.log_dir}/foreman-maintain.log")
       @logger = Logger.new(filename, 10, 10_240_000).tap do |logger|
-        logger.level = config.log_level || Logger::DEBUG
+        logger.level = LOGGER_LEVEL_MAPPING[config.log_level] || Logger::DEBUG
         logger.datetime_format = '%Y-%m-%d %H:%M:%S%z '
       end
     end
+
+    def load_custom_configs(file_path)
+      file_path ||= ''
+      custom_configs = {}
+      if File.exist?(file_path)
+        custom_configs = YAML.load(File.open(file_path)) || {}
+      else
+        $stderr.puts "Config file #{file_path} not found, using default configuration"
```

Comment:
Could you think of changing the code in a way that this messages would be produced after the logger is initialized and producing it via a logger?